src/model/guild/member.rs

```diff
@@ -559,8 +559,10 @@ pub struct PartialThreadMember {
 
 /// A model representing a user in a Guild Thread.
 ///
-/// [Discord docs](https://discord.com/developers/docs/resources/channel#thread-member-object),
-/// [extra fields](https://discord.com/developers/docs/topics/gateway-events#thread-member-update-thread-member-update-event-extra-fields).
+/// [Discord docs], [extra fields].
+///
+/// [Discord docs]: https://discord.com/developers/docs/resources/channel#thread-member-object,
+/// [extra fields]: https://discord.com/developers/docs/topics/gateway-events#thread-member-update-thread-member-update-event-extra-fields
```

Comment:
Is this not the same but longer?

Reply:
Kind of, but since the links are so long it's better not to have them inline as the text of the doc comment is less readable in code otherwise.

Reply:
Fair enough.